apps/bt510/tiltad/tiltad.py

```diff
@@ -1,3 +1,6 @@
+app_id='xbit_510_tiltad'
```

Comment:
Note there is code below that already sets the app_id and app_ver. You'll need to move your preferred version info down there. I recommend using 1.5.0 for the version as there have been tiltad.py script versions floating around for 6+ months now with older version #s.